imports/ui/components/sidebar_menu/PanelProfile.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PanelHeader from '../common/PanelHeader.jsx'
+import Nestable from '../common/Nestable.jsx'
+import PanelListItem from '../common/PanelListItem.jsx'
+import IconButton from '../common/IconButton.jsx'
+import LogoutAccount from '../login/LogoutAccount.jsx'
+import * as constants from '../common/Constants.js';
+
+export default class PanelProfile extends React.Component {
+  constructor(){
+    super();
+  }
+
+  render() {
+    return (
+      <nav className="side-menu-addl" style={styles.PanelProfile}>
+        <PanelHeader  title="User Profile" icon="font-icon font-icon-user" />
+
```

Comment:
Understood that these test values will be changed to utilize logic's getUserProfile methods!! @walrys